adcc/modified_transition_moments.py

```diff
@@ -72,10 +74,39 @@ def compute_modified_transition_moments(gs_or_matrix, dipole_operator,
                         "libadcc.OneParticleOperator.")
 
     if method.is_core_valence_separated:
-        raise NotImplementedError("compute_modified_transition_moments "
-                                  "not implemented for CVS")
-
+        return modified_transition_moments_cvs(gs_or_matrix,
+                                               dipole_operator, method)
     mtm_cpp = libadcc.compute_modified_transition_moments(
         method.name, ground_state, dipole_operator
     )
     return AmplitudeVector(*mtm_cpp.to_tuple())
+
+
+def modified_transition_moments_cvs(gs_or_matrix, dipole_operator,
+                                    method=None):
+    if isinstance(gs_or_matrix, AdcMatrix):
+        ground_state = gs_or_matrix.ground_state
+        intermediates = gs_or_matrix.intermediates
+    elif isinstance(gs_or_matrix, LazyMp):
+        ground_state = gs_or_matrix
+        intermediates = AdcMatrix(method, ground_state).intermediates
```

Comment:
That's a bit of a quirk :smile:. Just construct the `AdcIntermediates` class, see https://github.com/adc-connect/adcc/blob/master/adcc/state_densities.py#L47.

Reply:
I'm learnding

Reply:
Sure, that's what reviews are for. I hope I don't come across as too harsh ... not my intention.